bin/extract-assets

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -e
+
+docker cp pattern-library-gulp-watch:/srv/pattern-library/source/assets .container_source_assets
```

Comment:
Could there be an option to run this as a watch task to keep `.container_source_assets` up to date with whatever's generated within the container?

Reply:
The key thing here is separating writes performed by the container from the source owned by the host, so I'll try to change this into a mounted folder.